test/rotoscope_test.rb

```diff
@@ -145,9 +298,9 @@ def assert_frames_consistent(csv_string)
     assert_equal csv_string.scan(/\Acall/).size, csv_string.scan(/\Areturn/).size
   end
 
-  def rotoscope_trace(blacklist = [])
-    Rotoscope.trace(@logfile, blacklist) { |rotoscope| yield rotoscope }
-    unzip(@logfile)
+  def rotoscope_trace(*args)
+    Rotoscope.trace(@logfile, *args) { |rotoscope| yield rotoscope }
+    File.open(@logfile).read
```

Comment:
🤦‍♂️ Yeah I realized this when I was writing it but then promptly forgot to change it.